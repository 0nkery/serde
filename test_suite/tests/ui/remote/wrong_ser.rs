#[macro_use]
extern crate serde_derive;

mod remote {
    pub struct S {
        pub a: u16,
    }
}

#[derive(Serialize)]
#[serde(remote = "remote::S")]
struct S {
    a: u8,
}

fn main() {}
