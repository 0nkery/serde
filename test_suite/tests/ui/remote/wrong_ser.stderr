error[E0308]: mismatched types
  --> $DIR/wrong_ser.rs:10:10
   |
10 | #[derive(Serialize)]
   |          ^^^^^^^^^ expected u8, found u16
   |
   = note: expected type `&u8`
              found type `&u16`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0308`.
